docs/usage/peer.md

```diff
@@ -17,6 +17,8 @@ The following sections present the respective procedures to **peer a local clust
 At the end of the process, a new **virtual node** is created in the consumer, abstracting the resources shared by the provider, and enabling seamless **pod offloading** to the remote cluster.
 Additional details are also provided to enable the reverse peering direction, hence achieving a **bidirectional peering**, allowing both clusters to offload a part of their workloads to the other.
 
+By default, Liqo shares a fixed percentage of the available resources of the origin cluster. You can change this behaviour by using a [resource plugin](https://github.com/liqotech/liqo-resource-plugins).
```

Comment:
```suggestion
By default, Liqo shares a configurable percentage of the currently available resources of the *provider* cluster with *consumers*. 
You can change this behavior by using a custom [resource plugin](https://github.com/liqotech/liqo-resource-plugins).
```

I wouldn't say *fixed*, since it dynamically varies based on the currently available resources.